ticdc/ticdc-sink-to-cloud-storage.md

```diff
@@ -59,24 +59,83 @@ For `[query_parameters]` in the URI, the following parameters can be configured:
+    - `s3:ListMultipartUploadParts`
+    - `s3:ListBucketMultipartUploads`
+
+If you have not created a replication data storage directory, you can refer to [Create a Bucket](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/create-bucket.html) to create an S3 storage bucket in the specified region. If you need to use a folder, you can refer to [Organizing objects in the Amazon S3 console by using folders](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/create-folder.html) to create a folder in the storage bucket.
+
+You can configure an account to access Amazon S3 in either of the following two ways:
+
+- Method 1: Specify the access key
+
+    If you specify an access key and a secret access key, authentication will be performed according to them. In addition to specifying the key in the URI, the following methods are supported:
+
+    - Read the `$AWS_ACCESS_KEY_ID` and `$AWS_SECRET_ACCESS_KEY` environment variables
+    - Reading the `$AWS_ACCESS_KEY` and `$AWS_SECRET_KEY` environment variables
+    - Read the shared credentials file, with the path specified by the `$AWS_SHARED_CREDENTIALS_FILE` environment variable
+    - Read the shared credentials file, with the path `~/.aws/credentials`
+
+- Method 2: Access based on an IAM role
+
+    Associate an [IAM role with configured permissions to access Amazon S3 ](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2.html) to the EC2 instance running TiCDC server. After successful setup, TiCDC can directly access the corresponding backup catalog in Amazon S3 without additional setup.
+
+</div>
+<div label="GCS" value="gcs">
+
 The following is an example configuration for GCS:
 
 ```shell
 --sink-uri="gcs://bucket/prefix?protocol=canal-json"
 ```
 
+You can configure the accounts that have access to GCS by specifying an access key. Authentication is performed according to the specified `credentials-file`. In addition to specifying the key file in the URI, the following methods are supported:
+
+- Read the contents of a file located in the path specified by the `$GOOGLE_APPLICATION_CREDENTIALS` environment variable.
+- Retrieve the contents of the file in `~/.config/gcloud/application_default_credentials.json
+- Get credentials from a metadata server when running in GCE or GAE
+
+</div>
+<div label="Azure Blob Storage" value="azure">
+
 The following is an example configuration for Azure Blob Storage:
 
 ```shell
 --sink-uri="azure://bucket/prefix?protocol=canal-json"
 ```
 
+You can configure the accounts that access Azure Blob Storage in the following ways:
+
+- Method 1: Specify a shared access signature
+
+    If you configure `account-name` and `sas-token` in the URI, the storage account name and shared access signature token specified by this parameter are used. Because the shared access signature token has the `&` character, you need to encode it as `%26` before adding it to the URI. You can also just encode the entire `sas-token` in percent.
```

Comment:
```suggestion
    If you configure `account-name` and `sas-token` in the URI, the storage account name and shared access signature token specified by this parameter are used. Because the shared access signature token has the `&` character, you need to encode it as `%26` before adding it to the URI. You can also directly encode the entire `sas-token` using percent-encoding.
```